common/config.py

```diff
@@ -7,8 +7,14 @@
 
 # Environment variable-based configuration of benchmarking pandas or cudf.
 if "CUDF_BENCHMARKS_USE_PANDAS" in os.environ:
-    import pandas as cudf
     import numpy as cupy
+    import pandas as cudf
 else:
-    import cudf  # noqa: W0611
-    import cupy  # noqa: W0611
+    import cudf  # noqa: W0611, F401
+    import cupy  # noqa: W0611, F401
```

Comment:
Yeah it's a pylint error code. I think I added it to make my vim Python plugin happy because it includes pyflakes, but it won't cause errors for the pre-commit hooks. I can remove the noqa if you'd like.